api/api-cache.php

```diff
@@ -0,0 +1,177 @@
+<?php
+require_once('../template/top.php');
+
+// RegEx to match certain endpoints we want to ignore, e.g., orders
+const IGNORED_ENDPOINTS = '/(?:orders)/i';
```

Comment:
There's no need to have ignored endpoints, each entry in the db table would be a specific endpoint so if we want to ignore it we just won't make an entry for it.

Reply:
I think maybe the confusion here is that you have the code adding a cache entry automatically if its not found, we should not do this. Cache details need to be manually added to the database, if they don't exist and the retrieve cache method is called for that endpoint, we should just throw an error.

If we allow any call to create a cache entry, we can't enforce sensible TTLs, we'll get erroneously created entries, each time an endpoint or API base changes then suddenly we'll get duplicate config entries. We should stay on top of these things manually.

Reply:
Adjusted in c289fad, specifically, [here (line 73)](https://github.com/untrobotics/website/pull/158/commits/c289fad3a479b93eff024e7c5094ab293f1e3a7b#diff-970b238464763356bddea5f27fef2cba1f6162b327686a4fe165921e8bb448e9R73) and [here (line 87)](https://github.com/untrobotics/website/pull/158/commits/c289fad3a479b93eff024e7c5094ab293f1e3a7b#diff-970b238464763356bddea5f27fef2cba1f6162b327686a4fe165921e8bb448e9R87)